binplace.targets

```diff
@@ -50,7 +50,10 @@
     <BinPlaceConfiguration Condition="'$(BinPlaceILCInputFolder)' == 'true'" Include="uapaot-$(_bc_OSGroup)">
       <RuntimePath>$(ILCFXInputFolder)</RuntimePath>
     </BinPlaceConfiguration>
-
+    <!-- And the UAP folder for the F5 (CoreCLR UAP-debugging) scenario -->
+    <BinPlaceConfiguration Condition="'$(BinPlaceTestSharedFramework)' == 'true'" Include="uap-$(OSGroup)">
```

Comment:
We are doing this because we want to drop a couple of other things into the same folder, in particular the Runner and Launcher.

The structure is going to look something like this:
testhost\uap-Windows_NT-Debug-x64:
  - Launcher
  - Runner
  - shared
  - uap-f5-layout